pkg/util/types/types.go

```diff
@@ -73,6 +78,36 @@ func (a AWSStringSlice) Hash() *string {
 	return aws.String(output)
 }
 
+// Get the configmap that holds the whitelisted internet facing
+func GetInternetFacingConfigMap(namespace, ingressName string) map[string]string {
```

Comment:
If my understanding of the design (see my comment on `configuration.md`) is that we expect individual to name namespace as so:

```
data:
  mynamespace: my-ingress-name, 
  mynamespace: my-ingress-name2
  myothernamespace: my-other-ingress-name
```

I'm uncertain how the returned `map[string]string` here would function.

In that being captured in a map, you would end up with duplicate keys. Specifically, when there are multiple ingresses in the same namespace you wish to allow the `internet-facing` scheme. ( see example above ).

So you'd have a resulting conflict such as https://play.golang.org/p/4LhZeJBhOTX, in our case, i'm guessing the latter key would simply overwrite the former.

Were you able to validate a scenario with multiple ingresses in the same namespace?